ambari-server/src/main/resources/common-services/KAFKA/0.8.1/package/scripts/setup_ranger_kafka.py

```diff
@@ -84,14 +84,14 @@ def setup_ranger_kafka():
     if params.stack_supports_core_site_for_ranger_plugin and params.enable_ranger_kafka and params.kerberos_security_enabled:
       if params.has_namenode:
         Logger.info("Stack supports core-site.xml creation for Ranger plugin and Namenode is installed, creating create core-site.xml from namenode configurations")
-        setup_core_site_for_required_plugins(component_user = params.kafka_user, component_group = params.user_group,
+        setup_configuration_file_for_required_plugins(component_user = params.kafka_user, component_group = params.user_group,
                                              create_core_site_path = params.conf_dir, configurations = params.config['configurations']['core-site'],
-                                             configuration_attributes = params.config['configuration_attributes']['core-site'])
+                                             configuration_attributes = params.config['configuration_attributes']['core-site'], file_name='core-site.xml')
```

Comment:
Need to change `config['configuration_attributes']` to `config['configurationAttributes']`

Reply:
updated patch to address review comment.